src/bindings/python/tests_compatibility/test_ngraph/test_create_op.py

```diff
@@ -1055,7 +1056,7 @@ def test_interpolate(dtype):
 
     image_node = ng.parameter(image_shape, dtype, name="Image")
 
-    node = ng.interpolate(image_node, output_shape, attributes)
+    node = ng_opset1.interpolate(image_node, output_shape, attributes)
```

Comment:
Since I added a new opset with new version of interpolate operator, I have to specify opset version in the test that validates old operator version (from now on new version is default, so `ng.interpolate` would return new version of the operator).

Reply:
Maybe additional test is required to validate old version and keep this one as validates the default one?
Test name not specify that this test is for old version of operator.

Reply:
The version of tested op has been added. Now it shouldn't be confusing.